server/eclipse-project/src/main/java/us/freeandfair/corla/asm/ASMTransition.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Free & Fair Colorado RLA System
+ * 
+ * @title ColoradoRLA
+ * @created Aug 10, 2017
+ * @copyright 2017 Free & Fair
+ * @license GNU General Public License 3.0
+ * @author Daniel M. Zimmerman <dmz@freeandfair.us>
+ * @description A system to assist in conducting statewide risk-limiting audits.
+ */
+
+package us.freeandfair.corla.asm;
+
+import static us.freeandfair.corla.util.EqualsHashcodeHelper.nullableEquals;
+
+import java.io.Serializable;
+
+/**
+ * A single transition of an abstract state machine.
+ * 
+ * @author Daniel M. Zimmerman <dmz@freeandfair.us>
+ * @version 0.0.1
+ */
+public class ASMTransition implements Serializable {
+  /**
+   * The serialVersionUID.
+   */
+  private static final long serialVersionUID = 1L;
+
+  /**
+   * The start state for this transition.
+   */
+  private final ASMState my_start_state;
+  
+  /**
+   * The event for this transition.
+   */
+  private final ASMEvent my_event;
+  
+  /**
+   * The end state for this transition.
+   */
+  private final ASMState my_end_state;
+  
+  /**
+   * Constructs an AsmTransition with the specified start state,
```

Comment:
s/AsmTransition/ASMTransition. May want to check other comments of renamed classes.